clang/test/Driver/sycl-deprecated.cpp

```diff
@@ -11,3 +11,8 @@
 // RUN: | FileCheck %s --check-prefix=CHECK_REPLACE -DOPTION=-fno-sycl-use-bitcode -DOPTION_REPLACE=-fsycl-device-obj=spirv
 // RUN: %clangxx -fsycl -fsycl-fp32-prec-sqrt %s -### 2>&1 | FileCheck %s --check-prefix=CHECK_REPLACE -DOPTION=-fsycl-fp32-prec-sqrt -DOPTION_REPLACE=-foffload-fp32-prec-sqrt
 // CHECK_REPLACE: option '[[OPTION]]' is deprecated and will be removed in a future release, use '[[OPTION_REPLACE]]' instead
+
+// RUN: %clangxx -fsycl -fsycl-device-lib-jit-link %s -### 2>&1 \
+// RUN: | FileCheck %s --check-prefix=CHECK -DOPTION=-fsycl-device-lib-jit-link
+// RUN: %clangxx -fsycl -fno-sycl-device-lib-jit-link %s -### 2>&1 \
+// RUN: | FileCheck %s --check-prefix=CHECK -DOPTION=-fno-sycl-device-lib-jit-link
```

Comment:
Please add tests for `-fno-sycl-device-lib` and `-fsycl-device-lib` deprecation as well.

Reply:
Done.
Thanks very much.